### ISSUE TITLE
[feature] Finish full AIOps orchestrator end-to-end and make test-ready

### ISSUE THREAD

**Issue by ysbc1247** [capston-design-2026-1/Borg-Agent-Orchestrator#15]
## Task
Complete the full architecture end-to-end and leave the repository in a test-ready state.

Mandatory architecture layers to validate and integrate:
1) Local source and trace ingestion path
2) AIOpsLab-style simulator layer and feature extraction
3) XGBoost safety/resource predictors integrated into observation space
4) Ray RLlib PPO multi-agent layer (Agent A/B/C) with referee conflict logic
5) Optuna meta-optimizer tuning reward weights and RL hyperparameters
6) Global scoreboard and full feedback loop

Implementation and quality requirements:
- Run the full process command and fix breakages until a green run is achieved.
- Keep strict commit policy: near per-file or per-feature-slice commits.
- Push all commits.
- Keep AGENTS/NEXT_STEPS/README/report files synchronized with actual behavior.
- If gaps are found, create follow-up tasks/issues and link them.

Validation requirements:
- Ensure one-command testability for the full flow.
- Verify generated outputs/artifacts paths and write concise artifact descriptions.
- Confirm Ray + Optuna integration is active in execution, not placeholder code.

Completion criteria:
- The main full-process command runs successfully with realistic config.
- Docs include exact startup/test commands and troubleshooting notes.
- Remaining known risks are documented with concrete next actions.

- task_id: `full-orchestrator-e2e-finish`
- task_type: `feature`
- priority: `5`
- scope_paths: `orchestrator_stack/, codex_autonomy/, reports/, README.md, NEXT_STEPS.md`
- done_when: `./.venv/bin/python orchestrator_stack/run.py full-process --config orchestrator_stack/config/orchestrator.example.json --trials 1`
